modules/installation-configuration-parameters.adoc

```diff
@@ -1246,6 +1246,18 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 |The GCP service account used for the encryption request for control plane and compute machines. If absent, the Compute Engine default service account is used. For more information about GCP service accounts, see Google's documentation on link:https://cloud.google.com/compute/docs/access/service-accounts#compute_engine_service_account[service accounts].
 |The GCP service account email, for example `<service_account_name>@<project_id>.iam.gserviceaccount.com`.
 
+|`platform.gcp.defaultMachinePlatform.secureBoot`
+|Whether to enable Shielded VMs for all machines in the cluster. Shielded VMs have additional security protocols such as secure boot, firmware and integrity monitoring, and rootkit protection. For more information on Shielded VMs, see Google's documentation on link:https://cloud.google.com/shielded-vm[Shielded VMs].
+|`Enabled` or `Disabled`. The default value is `Disabled`.
+
+|`platform.gcp.defaultMachinePlatform.confidentialCompute`
+|Whether to enable Confidential VMs for all machines in the cluster. Confidential VMs provide encryption for data while it is being processed. For more information on Confidential VMs, see Google's documentation on link:https://cloud.google.com/confidential-computing[Confidential Computing].
```

Comment:
We can say `Enable confidential computing` or `use confidential VMs` , I'm not sure about  `enable confidential VMs`
```suggestion
|Whether to use Confidential VMs for all machines in the cluster. Confidential VMs provides encryption for data while it is being processed. For more information on Confidential computing, see Google's documentation on link:https://cloud.google.com/confidential-computing[Confidential Computing].
```